astropy/io/misc/asdf/tags/unit/equivalency.py

```diff
@@ -0,0 +1,47 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+
+from astropy.units.equivalencies import Equivalency
+from astropy.units import equivalencies
+from astropy.units.quantity import Quantity
+from asdf.yamlutil import custom_tree_to_tagged_tree
+
+from astropy.io.misc.asdf.types import AstropyAsdfType
+
+
+class EquivalencyType(AstropyAsdfType):
+    name = "unit/equivalency"
+    types = [Equivalency]
+    requires = ['astropy']
+    version = '1.1.0'
```

Comment:
The version of this tag will actually end up being `1.0.0` since it's a completely new type. This will need to be reflected in the schema as well.